components/aws/sagemaker/deploy/component.yaml

```diff
@@ -0,0 +1,21 @@
+name: 'Sagamaker Model Deployer'
```

Comment:
Nit: Components are like functions, so it might make sense to name them starting with a verb. E.g. "Sagemaker - Deploy model"